pkgs/build-support/php/hooks/composer-repository-hook.sh

```diff
@@ -65,7 +65,16 @@ composerRepositoryBuildHook() {
 composerRepositoryCheckHook() {
     echo "Executing composerRepositoryCheckHook"
 
-    composer validate --no-ansi --no-interaction
+    if ! composer validate --strict --no-ansi --no-interaction --quiet; then
```

Comment:
Are you _sure_ that on mismatch it will not silently update the composer.lock and download a random/latest version of the dependencies?

Reply:
Yes 100% sure. It won't alter the reproducible behaviour since Composer basically download what's in `composer.lock` only.